dvc/exceptions.py

```diff
@@ -23,13 +28,15 @@ class OutputDuplicationError(DvcException):
     def __init__(self, output, stages):
         assert isinstance(output, str)
         assert all(hasattr(stage, "relpath") for stage in stages)
-        msg = (
-            "file/directory '{}' is specified as an output in more than one "
-            "stage: \n{}\n"
-            "This is not allowed. Consider using a different output name."
-        ).format(
-            output, "\n".join("\t{}".format(s.addressing) for s in stages)
-        )
+        if len(stages) == 1:
+            msg = "output '{}' is already specified on {}.".format(
```

Comment:
Maybe `in` instead of `on`?

Reply:
Also stage name should be wrapped in `'` to mark that it is generated. Same In messages below.

Reply:
we have `stage: 'stage name'` as `__str__`. So, no need to wrap here. :smile: